app/views/restaurant/index.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Restaurant Ufood
```

Comment:
Is this a site for a restaurant named Ufood or is it a service for many restaurants?

Reply:
its a service for many restaurants, it was hastily put up like this. will change this.